scripts/generate-calculate-release-dates-api-types.sh

```diff
@@ -0,0 +1,2 @@
+npx openapi-typescript https://calculate-release-dates-api-dev.hmpps.service.justice.gov.uk/v3/api-docs | sed "s/\"/'/g" | sed "s/;//g" > ../server/@types/calculateReleaseDates/index.d.ts
```

Comment:
So this has now been moved into `package.json`. So you can run `npm run generate-calculate-release-dates-api-types`

Reply:
I didn't appreciate that, I've removed